### PR TITLE
[issue-8142] upgrade apache libs due to cves

### PR DESCRIPTION
**Pull request by pjfanning**
## Description

https://github.com/apache/pinot/issues/8142

## Upgrade Notes
Does this PR prevent a zero down-time upgrade? (Assume upgrade order: Controller, Broker, Server, Minion)
* [ ] Yes (Please label as **<code>backward-incompat</code>**, and complete the section below on Release Notes)

Does this PR fix a zero-downtime upgrade introduced earlier?
* [ ] Yes (Please label this as **<code>backward-incompat</code>**, and complete the section below on Release Notes)

Does this PR otherwise need attention when creating release notes? Things to consider:
- New configuration options
- Deprecation of configurations
- Signature changes to public methods/interfaces
- New plugins added or old plugins removed
* [ ] Yes (Please label this PR as **<code>release-notes</code>** and complete the section on Release Notes)

### CODE REVIEW

**Review by richardstartin — APPROVED** [review]
👍🏻

File: pinot-plugins/pinot-input-format/pinot-thrift/src/test/java/org/apache/pinot/plugin/inputformat/thrift/ThriftRecordExtractorTest.java [hunk 1]
```diff
@@ -130,7 +131,12 @@ protected void createInputFile()
     }
 
     BufferedOutputStream bufferedOut = new BufferedOutputStream(new FileOutputStream(_tempFile));
-    TBinaryProtocol binaryOut = new TBinaryProtocol(new TIOStreamTransport(bufferedOut));
+    TBinaryProtocol binaryOut = null;
+    try {
+      binaryOut = new TBinaryProtocol(new TIOStreamTransport(bufferedOut));
+    } catch (TTransportException e) {
+      throw new IOException(e);
+    }
```
[the remark this answers is not in the document]
Comment (pjfanning) [reply pinot-plugins/pinot-input-format/pinot-thrift/src/test/java/org/apache/pinot/plugin/inputformat/thrift/ThriftRecordExtractorTest.java]: @richardstartin I updated the code you highlighted

**Comment by pjfanning** [comment]
@richardstartin thanks for reviewing this - would it be possible to merge it?

**Review by xiangfu0 — APPROVED** [review]
Thanks for fixing this!

**Pull request merged**